PWG/EMCAL/EMCALbase/AliAnalysisTaskEmcal.cxx

```diff
@@ -1324,6 +1324,7 @@ Bool_t AliAnalysisTaskEmcal::IsTriggerSelected(){
 Bool_t AliAnalysisTaskEmcal::CheckMCOutliers()
 {
   if (!fPythiaHeader || !fMCRejectFilter) return kTRUE;
+  AliDebugStream(1) << "Using custom outlier rejection" << std::endl;
```

Comment:
Can you please increase this debug level a bit? Otherwise, I think this will be called every MC event (?), which I think could make debug other issues difficult

Reply:
@raymondEhlers Well the original debug level for other AliDebugs in the function was 1 as well, but no problem, raised them all to 2. However I wouldn't like to raise them more.